sceneview/src/main/java/io/github/sceneview/node/Node.kt

```diff
@@ -695,15 +695,16 @@ open class Node(
      * @param upDirection The up direction will determine the orientation of the node around the direction
      * @param smooth Whether the rotation should happen smoothly
      */
-    fun lookAt(
+    open fun lookAt(
         targetPosition: Position,
         upDirection: Direction = Direction(y = 1.0f),
         smooth: Boolean = false
     ) {
+        // targetPosition is passed as eye so that the forward direction will be towards the target
```

Comment:
I think it will be better to use a common style of comments (before the `look...` call or inside the arguments).